shared/templates/file_permissions/oval.template

```diff
@@ -1,3 +1,19 @@
+{{%- if ALLOW_STRICTER_PERMISSIONS %}}
+    {{#
+        When allowing stricter permissions, the check looks for files/directories that
+        do not match the permissions (STATEMODE). The criterion is then negated to make
```

Comment:
Text suggestion:
`looks for files/directories that have any extra permission bit set.`

Reply:
Extra suggestion: `The permissions (STATEMODE) is negated (inverted?) and becomes a mask for permissions that should not be set.`